src/components/KeyboardAwareScrollView/index.tsx

```diff
@@ -38,6 +38,10 @@ export type KeyboardAwareScrollViewProps = {
   enabled?: boolean;
   /** Adjusting the bottom spacing of KeyboardAwareScrollView. Default is `0` */
   extraKeyboardSpace?: number;
+   /** Used if inverted Flatlist/Flashlist is being used */
+  inverted?: boolean;
```

Comment:
I think this property can be derived from `style` property?

```tsx
style?.transform?.some(({scaleY}) => scaleY === -1);
```

Reply:
Maybe I am missing how to call style, but when I call it as is in the component, I get 

"Cannot find name 'style'."

 and in rest.style I have only 
`"style": {"backgroundColor": "#FFFFFF", "flex": 1}}` so ni scaleY.

But this inverted prop should be ok because it is a Flatlist/Flashlist prop. Right?

Reply:
@lpatrun can you show me how do you use `FlatList` and `KeyboardAwareScrollView`?

I assumed it would be something like that:

```tsx
<FlatList
  data={[]}
  renderItem={() => <></>}
  inverted
  renderScrollComponent={(props) => <KeyboardAwareScrollView {...props} />}
/>
```

And here (if I understand correctly) `props.style` should contain `transform` property (it should be injected by `FlatList`). Or no? 🙈

Reply:
This is the code from the app

`
 <FlatList
                contentContainerStyle={styles.dataStyles}
                data={[...messages].reverse()}
                inverted
                renderItem={({ item }) => renderChatElement(item)}
                ItemSeparatorComponent={() => <SpacingComponent val={4} />}
                renderScrollComponent={props => <KeyboardAwareScrollView {...props} />}
                ListHeaderComponent={() => (isLoading ? <Loading /> : <></>)}
            />
 `

Reply:
Thanks @lpatrun 

I'll check it today! ❤️

Reply:
Hi @lpatrun 

I added this code to `KeyboardAwareScrollView` (it has TS errors):

```tsx
    const inverted = Array.isArray(rest.style)
      ? rest.style?.[0]?.transform[0]?.scaleY === -1
      : false;

    console.log("rest", rest.style, inverted);
```

And this code I used for testing:

```tsx
    <FlatList
      // contentContainerStyle={{flex: 1}}
      // style={{flexGrow: 1}}
      data={[1]}
      inverted
      renderItem={({ item }) => <></>}
      renderScrollComponent={props => <KeyboardAwareScrollView {...props} />}
  />
```

And it seems to work fine:

<img width="560" alt="image" src="https://github.com/user-attachments/assets/2b027468-e37a-4960-adce-c07213ca4e4e">

Can you check on your side again?